t/path.t

```diff
@@ -27,265 +76,449 @@ CPAN/authors/id/J/JA/JAMCC/ngb-101.zip
 	extension	tar.gz
+	pkgurl		pkg:cpan/NI-S/Tk@400.202
 
 authors/id/G/GB/GBARR/perl5.005_03.tar.gz
 	filename	perl5.005_03.tar.gz
 	dist		perl
 	maturity	released
 	distvname	perl5.005_03
 	version		5.005_03
+	fullversion	5.005_03
 	cpanid		GBARR
 	extension	tar.gz
+	pkgurl		pkg:cpan/GBARR/perl@5.005_03
 
 M/MS/MSCHWERN/Test-Simple-0.48_01.tar.gz
 	filename	Test-Simple-0.48_01.tar.gz
 	dist		Test-Simple
 	maturity	developer
 	distvname	Test-Simple-0.48_01
 	version		0.48_01
+	fullversion	0.48_01
 	cpanid		MSCHWERN
 	extension	tar.gz
+	pkgurl		pkg:cpan/MSCHWERN/Test-Simple@0.48_01
 
 id/J/JV/JV/PostScript-Font-1.09.tar.gz
 	filename	PostScript-Font-1.09.tar.gz
 	dist		PostScript-Font
 	maturity	released
 	distvname	PostScript-Font-1.09
 	version		1.09
+	fullversion	1.09
 	cpanid		JV
 	extension	tar.gz
+	pkgurl		pkg:cpan/JV/PostScript-Font@1.09
 
 id/I/IB/IBMTORDB2/DBD-DB2-0.77.tar.gz
 	filename	DBD-DB2-0.77.tar.gz
 	dist		DBD-DB2
 	maturity	released
 	distvname	DBD-DB2-0.77
 	version		0.77
+	fullversion	0.77
 	cpanid		IBMTORDB2
 	extension	tar.gz
+	pkgurl		pkg:cpan/IBMTORDB2/DBD-DB2@0.77
 
 id/I/IB/IBMTORDB2/DBD-DB2-0.99.tar.bz2
 	filename	DBD-DB2-0.99.tar.bz2
 	dist		DBD-DB2
 	maturity	released
 	distvname	DBD-DB2-0.99
 	version		0.99
+	fullversion	0.99
 	cpanid		IBMTORDB2
 	extension	tar.bz2
+	pkgurl		pkg:cpan/IBMTORDB2/DBD-DB2@0.99?ext=tar.bz2
 
 CPAN/authors/id/L/LD/LDS/CGI.pm-2.34.tar.gz
 	filename	CGI.pm-2.34.tar.gz
 	dist		CGI
 	maturity	released
 	distvname	CGI.pm-2.34
 	version		2.34
+	fullversion	2.34
 	cpanid		LDS
 	extension	tar.gz
+	pkgurl		pkg:cpan/LDS/CGI.pm@2.34
+	SKIP		Dist name with .pm found only on BackPAN
 
 CPAN/authors/id/J/JE/JESSE/perl-5.12.0-RC0.tar.gz
 	filename	perl-5.12.0-RC0.tar.gz
 	dist		perl
 	maturity	developer
 	distvname	perl-5.12.0-RC0
 	version		5.12.0-RC0
+	fullversion	5.12.0-RC0
 	cpanid		JESSE
 	extension	tar.gz
+	pkgurl		pkg:cpan/JESSE/perl@5.12.0-RC0
+	NOTE		perl distribution version may include RC\d* suffix
 
 CPAN/authors/id/G/GS/GSAR/perl-5.6.1-TRIAL3.tar.gz
 	filename	perl-5.6.1-TRIAL3.tar.gz
 	dist		perl
 	maturity	developer
 	distvname	perl-5.6.1-TRIAL3
 	version		5.6.1-TRIAL3
+	fullversion	5.6.1-TRIAL3
 	cpanid		GSAR
 	extension	tar.gz
+	pkgurl		pkg:cpan/GSAR/perl@5.6.1-TRIAL3
+	NOTE		perl distribution version may include TRIAL\d* suffix
 
 CPAN/authors/id/R/RJ/RJBS/Dist-Zilla-2.100860-TRIAL.tar.gz
 	filename	Dist-Zilla-2.100860-TRIAL.tar.gz
 	dist		Dist-Zilla
 	maturity	developer
 	distvname	Dist-Zilla-2.100860-TRIAL
-	version		2.100860-TRIAL
+	version		2.100860
+	fullversion	2.100860-TRIAL
 	cpanid		RJBS
 	extension	tar.gz
+	pkgurl		pkg:cpan/RJBS/Dist-Zilla@2.100860-TRIAL
 
 CPAN/authors/id/M/MI/MINGYILIU/Bio-ASN1-EntrezGene-1.10-withoutworldwriteables.tar.gz
 	filename	Bio-ASN1-EntrezGene-1.10-withoutworldwriteables.tar.gz
 	dist		Bio-ASN1-EntrezGene
 	maturity	released
 	distvname	Bio-ASN1-EntrezGene-1.10-withoutworldwriteables
 	version		1.10
+	fullversion	1.10-withoutworldwritables
```

Comment:
Typo in "fullversion" (`1.10-withoutworldwritables` --> `1.10-withoutworldwriteables`)